test-data/unit/diff.test

```diff
@@ -618,3 +618,53 @@ p = Point(dict(x=42, y=1337))
 [out]
 __main__.Point
 __main__.p
+
+[case testTypeAliasSimple]
+A = int
```

Comment:
In all of these test cases, also include a case where there is no difference to make sure that we don't just always consider type aliases different. Example:

```
[case testTypeAliasSimple]
A = int
B = int
[file next.py]
A = str
B = int
[out]
__main__.A
```